README.md

```diff
@@ -106,6 +107,15 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 There are a range of options for installing the conda plugin. You'll need to build it first, and
 then install the plugin with the options shown below:
 
+### Easiest Install (beta)
+
+Thanks to some upstream work in Nexus Repository (versions newer than 3.15), it's become a LOT easier to install a plugin. To install the `conda` plugin, follow these steps:
+
+* Build the plugin with `mvn clean package -PbuildKar`
+* Copy the `nexus-repository-conda-0.0.1-bundle.kar` file from your `target` folder to the `deploy` folder for your Nexus Repository installation.
+
+If everything goes ok, within a minute or so the plugin should install itself, and you'll be able to start using it in Nexus Repository.
```

Comment:
I confirmed that "hot load" of a .kar does not work fully in that the "config" screen will not be shown when you click on the recently loaded format in the list.

I chatted with Stuart about this, and while the UI contributions will 'hot load', "the repository facet has its own cache and would need to be restarted for that part to be functional -  in 3.16 these SERVICES can be bounced by using REST or the console without needing to restart the whole process. ...generally it's safer to copy the kar over and then start nxrm"

That said, I think these instructions should be changed to explicitly start/restart NXRM after copying the .kar file to the `deploy` folder.

Reply:
Sounds good! Thanks for giving it a try!